### DISCUSSION TITLE
i created a beatmap on osu!lazer but i want to upload it on the website.

### DISCUSSION THREAD

**Discussion by shinmorisawa** [ppy/osu#24355] [Ideas] [answered]
i created a beatmap on osu!lazer but i want to upload it on the website. i was wondering if this was possible.

**Comment by peppy** [comment, marked as answer]
Please wait for the next release, which will add a better export option from the editor. You can then use osu-stable to upload your beatmap.

Proper upload support from the lazer editor will come a bit later.
